packages/pass-style/package.json

```diff
@@ -19,6 +19,7 @@
   "exports": {
     ".": "./index.js",
     "./tools.js": "./tools.js",
+    "./endow.js": "./endow.js",
```

Comment:
What is the purpose of splitting the symbol in its own external path? From what I gather it doesn't help with any kind of module shaking because all modules are still necessarily imported.

Reply:
Only separation of concerns. Clients of the package, in the normal sense, would import the API in `index.js`. Only providers of (some of) the API would import the API is `endow.js`. This does not help security, since it is a registered symbol and therefore forgeable. This does not help space as you say. The intention is only separation of concerns.

Reply:
I'm not convinced of the need. The endower need to create `passStyleOf` in the first place, so I would expect them to import from `index.js`

Reply:
It's not to keep the endower's view simpler. It is to keep the normal client's view simpler.

Reply:
The normal client should be unaware of the existence of this symbol. The normal client should see no change if we switch to a different endowing mechanism.

Reply:
I see. I had missed that index.js was not wholesale re-exporting passStyleOf.js